Gemfile

```diff
@@ -47,6 +47,10 @@ gem 'aws-sdk', '~> 2'
 # logger for debugging AWS
 gem 'logger'
 
+# # Caching for production
+# gem 'rack-cache'
+# gem 'redis-rack-cache'
```

Comment:
Heh, this is just what I had locally. I think it's a good idea to keep them in commented to highlight that caching is disabled on production. It's also an open issue I on the repository.